src/themes/teams/components/Icon/svg/icons/callControlPresentNew.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { IconSpec } from '../types'
+
+export default (classes => (
+  <svg viewBox="0 0 32 32" role="presentation">
+    <path
+      d="M22.5,9h-13C8.672852,9,8,9.672852,8,10.5v9C8,20.327148,8.672852,21,9.5,21H13c0.276367,0,0.5-0.223633,0.5-0.5
```

Comment:
Should we pass the classes to the svg element and the paths here? Is this where the path1, path2 etc. objects from the iconStyles will be used?

Reply:
>  Is this where the path1, path2 etc. objects from the iconStyles will be used?

yes, exactly. I would leave this work to icons' maintainers, as well as making final decision on how styling parts should be named for icons. Once we'll agree that provided mechanisms are sufficient enough to meet their needs, the task will reduce only to styling aspects that I would rather prefer to be addressed by design folks